esp/esp/cache/apps.py

```diff
@@ -1,14 +1,10 @@
-from django.apps import AppConfig
+from argcache.apps import ArgCacheConfig
 
-class CacheConfig(AppConfig):
+class ESPCacheConfig(ArgCacheConfig):
     """ Makes sure all caches are inserted. Replacement for cache_loader. """
 
-    name = 'esp.cache'
-
     def ready(self):
-        import sys
-        from django.conf import settings
-        from esp.cache.registry import _finalize_caches, _lock_caches
+        from argcache.registry import _finalize_caches, _lock_caches
```

Comment:
No, and I put these to the toplevel in the new non-ESP-specific version of this file (https://github.com/luac/django-argcache/blob/master/src/apps.py).